leaderless/node/app/main.py

```diff
@@ -4,8 +4,11 @@
 
 from app.database import *
 
+from leaderless.node.app.acceptor import Acceptor
```

Comment:
I don't think this will compile. Other imports look different

Reply:
Fixed